docs/components/store.md

```diff
@@ -27,7 +27,8 @@ In general about 1MB of local disk space is required per TSDB block stored in th
 
 ## Flags
 
-[embedmd]:# (flags/store.txt $)
+[embedmd]: # "flags/store.txt $"
```

Comment:
changes done by vscode automatically, please advice if I should remove this or not

Reply:
That's fine as long as embedding works.